runtime/jcl/j9jcl.tdf

```diff
@@ -210,7 +210,7 @@ TraceEntry=Trc_JCL_com_ibm_oti_shared_getCachedToken_Entry Overhead=1 Level=1 Te
 TraceExit=Trc_JCL_com_ibm_oti_shared_getCachedToken_ExitFound Overhead=1 Level=1 Template="JCL: com.ibm.oti.shared getCachedToken: Exiting with found cached token"
 TraceExit=Trc_JCL_com_ibm_oti_shared_getCachedToken_ExitNull Overhead=1 Level=1 Template="JCL: com.ibm.oti.shared getCachedToken: Exiting with NULL"
 
-TraceEntry=Trc_JCL_com_ibm_oti_shared_getCachedURL_Entry Overhead=1 Level=2 Template="JCL: com.ibm.oti.shared getCachedURL: Entering for helperID=%d, cpeType=%d"
+TraceEntry=Trc_JCL_com_ibm_oti_shared_getCachedURL_Entry Overhead=1 Level=2 Template="JCL: com.ibm.oti.shared getCachedURL: Entering for helperID=%d"
```

Comment:
You can't change an existing tracepoint, a new one needs to be created, and this one can be marked Obsolete.

Reply:
https://github.com/eclipse-openj9/openj9/blob/master/doc/diagnostics/AddingTracepoints.md

The idea is that you should be able to use the latest JVM to format a Snap file from any older version.

Reply:
Thanks Peter. I updated the trace points.